cockatrice/src/remoteclient.cpp

```diff
@@ -150,6 +150,11 @@ void RemoteClient::loginResponse(const Response &response)
         if (resp.missing_features_size() > 0 && settingsCache->getNotifyAboutUpdates())
                 emit notifyUserAboutUpdate();
 
+        if (resp.server_features_size() > 0) {
+            for (int i = 0; i < resp.server_features_size(); ++i)
+                settingsCache->addServerFeatures(QString::fromStdString(resp.server_features(i)));
```

Comment:
Yeah, I'm not liking this being on `settingsCache` - that's for local configuration regardless of what server you're on